eqs/src/eth_polling.rs

```diff
@@ -365,6 +370,19 @@ impl EthPolling {
                     updated_state.last_reported_index = Some(current_index);
                     self.last_event_index = Some(current_index);
 
+                    //add assets to local map
+                    for wrap in wraps {
+                        match wrap {
+                            CapeTransition::Wrap { ro, src_addr, .. } => {
+                                updated_state
+                                    .address_from_assetdef
+                                    .insert(ro.asset_def, Address::from(src_addr));
+                            }
+                            CapeTransition::Faucet { .. } => {}
+                            CapeTransition::Transaction { .. } => {}
```

Comment:
If these are unreachable (assuming based on the name `wraps`) can we replace this with `_ => unreachable!()` or `panic!()` or something?